diff --git a/CMakeLists.txt b/CMakeLists.txt
index e091f3a..cb92a41 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,10 +57,10 @@ find_package(sensor_msgs REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(std_srvs REQUIRED)
 find_package(visualization_msgs REQUIRED)
-find_package(pcl_ros REQUIRED)
 find_package(pcl_conversions REQUIRED)
 find_package(livox_ros_driver2 REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
+find_package(tf2_ros REQUIRED)
 
 set(dependencies
   rclcpp
@@ -71,9 +71,9 @@ set(dependencies
   std_msgs
   std_srvs
   visualization_msgs
-  pcl_ros
   pcl_conversions
   livox_ros_driver2
+  tf2_ros
 )
 
 # Thirdparty libraries
diff --git a/package.xml b/package.xml
index 8c39e18..729bde8 100644
--- a/package.xml
+++ b/package.xml
@@ -25,7 +25,7 @@
   <depend>sensor_msgs</depend>
   <depend>common_interfaces</depend>
   <depend>tf2</depend>
-  <depend>pcl_ros</depend>
+  <depend>tf2_ros</depend>
   <depend>pcl_conversions</depend>
   <depend>livox_ros_driver2</depend>
 
